pkg/proxy/ipvs/graceful_termination.go

```diff
@@ -164,10 +164,10 @@ func (m *GracefulTerminationManager) deleteRsFunc(rsToDelete *listItem) (bool, e
 	}
 	for _, rs := range rss {
 		if rsToDelete.RealServer.Equal(rs) {
-			// Delete RS with no connections
-			// For UDP, ActiveConn is always 0
-			// For TCP, InactiveConn are connections not in ESTABLISHED state
-			if rs.ActiveConn+rs.InactiveConn != 0 {
+			// Delete TCP RS with no connections
+			// For other protocols, don't perform graceful termination
```

Comment:
Does this also apply for SCTP?

Reply:
I'm not familiar with SCTP so I chose to enable graceful termination for TCP only

I had a quick look at the protocol and it seems to include a handshake and a shutdown which should make connection tracking similar to TCP (so when a backend stops, the equivalent of a FIN is sent and the connection should be removed from the IPVS conntrack).

If you have a simple test scenario in mind I can look into it (the change is very simple we can simply do `rsToDelete.VirtualServer.Protocol != "udp"` instead)

Reply:
I'm not super familiar with it either, was more asking in case you knew :P. I think handling this on case-by-case basis makes sense. We can revisit if it's a problem for SCTP. Going to give another pass at this PR some time tomorrow :)

Reply:
I put more thought into this, and I _think_ to be safe, we should only follow this scenarios for protocols we know for sure run into issue (UDP). So `rsToDelete.VirtualServer.Protocol != "udp"` like you mentioned. Otherwise this could have unintended outcomes for sctp that we aren't aware of.

Reply:
i.e. don't change the behavior for sctp unless we have a good reason.

Reply:
Let's do that. Not sure how to test it but I'm pretty sure SCTP will have real connection tracking